File: exercicios/cpp/Instruções Sequenciais/Q15.cpp
/*
------------------------------------------------
15. Crie um algoritmo/fluxograma/programa que 
solicite ao usuário o raio e calcule e imprima 
o perímetro e a área do círculo correspondente. 
Usar a biblioteca cmath.
 */
#include <iostream>
#include <cmath>
#include <clocale>

using namespace std;
int main () {
	setlocale(LC_ALL,"Portuguese");
	double raio;
	int perimetro, area;
	cout<<"Informe o raio:";
	cin>>raio;
    perimetro = 2 * M_PI * raio;
    area = M_PI * pow(raio, 2);
    cout<<"Perímetro:"<<perimetro<<endl;
    cout<<"Área:"<<area<<endl;
	return 0;
}

File: exercicios/python/Instruções Sequenciais/Q15.py
# Exercício 15 — Perímetro e área do círculo
import math

def main():
    raio = float(input("Informe o raio do círculo: "))
    perimetro = 2 * math.pi * raio
    area = math.pi * raio**2
    print(f"Perímetro: {perimetro:.2f}")
    print(f"Área: {area:.2f}")

if __name__ == "__main__":
    main()